### PR TITLE
Add prometheus-snmp-exporter chart

### PR DESCRIPTION
**Pull request by Miouge1**
#### What this PR does / why we need it:
Add prometheus-snmp-exporter chart

#### Special notes for your reviewer:

Inspired from the prometheus-blackbox-exporter

#### Checklist
- [x] [DCO](https://www.helm.sh/blog/helm-dco/index.html) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md

### CODE REVIEW

**Comment by Miouge1** [comment]
@gianrubio and @mgoodness can you have a look at this?

File: stable/prometheus-snmp-exporter/templates/deployment.yaml [hunk 1]
```diff
@@ -0,0 +1,92 @@
+apiVersion: apps/v1beta2
```
Comment (davidkarlsen) [inline stable/prometheus-snmp-exporter/templates/deployment.yaml]: use v1

**Review by davidkarlsen — CHANGES_REQUESTED** [review]
Please also follow https://github.com/helm/helm/blob/master/docs/chart_best_practices/rbac.md
See https://github.com/helm/charts/blob/master/REVIEW_GUIDELINES.md for labels

Maybe some lines should be added to tell how to be able to scrape a given SNMP device? How to I make prometheus use this chart to scrape a given device's SNMP values?

File: stable/prometheus-snmp-exporter/templates/deployment.yaml (hunk 1 above)
Comment (Miouge1) [reply stable/prometheus-snmp-exporter/templates/deployment.yaml]: OK, fixed in latest commit.

**Comment by Miouge1** [comment]
@davidkarlsen thanks for the feedback, I didn't know about the new labels.

Have a look at the latest commit, i've added:
- RBAC and ServiceAccount
- new label format
- Documentation of Prometheus server configuration

**Review by davidkarlsen — APPROVED** [review]

**Pull request merged**